linode/resource_linode_nodebalancer_node_test.go

```diff
@@ -73,6 +73,17 @@ func TestAccLinodeNodeBalancerNode_update(t *testing.T) {
 					resource.TestCheckResourceAttr(resName, "weight", "200"),
 				),
 			},
+			{
```

Comment:
Can we make this a different test? Maybe `TestAccLinodeNodeBalancerNode_instanceRecreated`